.travis.yml

```diff
@@ -20,6 +20,7 @@ env:
 - ARGS="--resolver lts-3 --flag inline-c:gsl-example"
 - ARGS="--resolver lts --flag inline-c:gsl-example"
 - ARGS="--resolver nightly --flag inline-c:gsl-example"
+- ARGS="--resolver nightly"
```

Comment:
Maybe we should remove the previous line, with a comment on why the gsl example doesn't work with nightly?